sources/core/Stride.Core.Mathematics/Stride.Core.Mathematics.csproj

```diff
@@ -12,6 +12,7 @@
     <StrideAssemblyProcessorOptions>--auto-module-initializer --serialization</StrideAssemblyProcessorOptions>
     <StridePublicApi>true</StridePublicApi>
     <StrideCodeAnalysis>true</StrideCodeAnalysis>
+    <LangVersion>12.0</LangVersion>
```

Comment:
Is this still necessary?
cf https://github.com/stride3d/stride/pull/1616#discussion_r1372368581

Reply:
nope these were old net7 workarounds